packages/environment-check/src/checks/get-installed.ts

```diff
@@ -168,10 +166,13 @@ export async function getFioriGenVersion(): Promise<string> {
         const fioriGenPath = await getFioriGenGlobalPath();
         genSearchPaths.push(fioriGenPath);
 
-        if (isAppStudio()) {
-            const fioriGenPathBAS = join(globalNodeModulesBAS, NpmModules.FioriGenerator);
-            const fioriGenPathPnpmBAS = join(globalNodeModulePnpmBAS, NpmModules.FioriGenerator);
-            genSearchPaths.push(fioriGenPathBAS, fioriGenPathPnpmBAS);
+        // BAS recommend using NODE_PATH as the paths therein are used to locate all generators regardless of installation method
+        if (isAppStudio() && process.env.NODE_PATH) {
```

Comment:
Is `process.env.NODE_PATH` guaranteed to be set on BAS? If so, why check it? If not, what happens when it's not set?

Is there doco from BAS about this somewhere?

Reply:
I have added tests to cover it not being set. It should be set, but better to be safe. The only doc is emails. It was their suggestion.